eng/pipelines/libraries/stress/http.yml

```diff
@@ -45,8 +45,17 @@ jobs:
   
   - bash: |
       cd '$(httpStressProject)'
+      export HTTPSTRESS_CLIENT_ARGS="$HTTPSTRESS_CLIENT_ARGS -http 2.0"
+      export HTTPSTRESS_SERVER_ARGS="$HTTPSTRESS_SERVER_ARGS -http 2.0"
       docker-compose up --abort-on-container-exit --no-color
-    displayName: Run HttpStress
+    displayName: Run HttpStress - HTTP/2
```

Comment:
`HTTP/2` while https://github.com/dotnet/runtime/pull/41592/files#diff-53dfbdf75e79a04b57ab654be70b8cfcR86 has `HTTP 2.0` and all 1.1 have `HTTP 1.1`.

Reply:
It's now HTTP/2 and HTTP 1.1 which looks pretty standard.

Reply:
This line is `HTTP/2`, another line in the same file:
https://github.com/alnikola/runtime/blob/59e333e170b6e72f1e85f92928e1a072a1cd3f30/eng/pipelines/libraries/stress/http.yml#L86
`HTTP 2.0`

2 occurrences, both different. And it looks super weird in the pipeline:
![image](https://user-images.githubusercontent.com/11718369/91755958-3ef01a80-ebcc-11ea-82d7-6eba3d98f4b0.png)

We should either go with `/` or without in all occurrences (HTTP 1.1 as well as 2.0).

Reply:
I'm taking care of it in https://github.com/dotnet/runtime/pull/41513